### ISSUE TITLE
Prepare for the Lovly Buddy

### ISSUE THREAD

**Issue by eXtremozrh** [disdain13/PokeRoadie#188]
in the new update u can choose a Buddy at your shoulder. every Km it gives u a Candy.

Would absolutly like if we can choos, a buddy , when the update comes...

Rock it!

**Comment by adr-sit** [comment]
Anyone tried the bot with the Buddy on?

**Comment by Magyar00** [comment]
Yep, doesn't change anything, you juste have to put your buddy on (by your phone), run the bot, and then when you come back to your appli on your phone your Buddy has collected some candies

**Comment by ArthurJS** [comment]
Would love to see Buddy support
